react/modules/OrderItemsContext.ts

```diff
@@ -1,21 +1,44 @@
 import { useContext, createContext } from 'react'
 
+export interface AddItemsOptions {
+  allowedOutdatedData?: string[]
+  marketingData?: Partial<MarketingData>
+  salesChannel?: string
+}
+
+export interface UpdateItemOptions {
+  allowedOutdatedData?: string[]
+}
```

Comment:
Do you think we can sync this with GraphQL? Just so we won't have to manually update it every time something in the schema changes.
I suppose there will be some problems due to `Maybe` though, and I'm not sure what is the best way to solve it :-/

Reply:
I don't know if it is possible because it not only the types we need to update when we add something new in the schema, but other parts of the code need to change as well, so we would need to change this app anyway

Reply:
I see. So do you believe it should be fixed, but not now, right?

Reply:
yeah, probably would need to reestructure some parts of the code to make it work when we only update the GraphQL